### ISSUE TITLE
Clear CSS rules this site shows no UI indication to reload page 

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#903]
## Description 

Clear CSS rules for page should auto reload the page

## Steps to Reproduce

   1. Visit any page and block an element via context menu, element is removed immediately from view
   2. Right click and clear CSS rule for this site
   3. No UI indication to reload the page to bring back the unblocked element

## Actual result:

Clear CSS rule for this site shows no UI indication to reload the page

## Expected result:
Clear CSS rule for this site should **trigger an auto reload** to unhide the element or should show some UI indication to reload the page

## Reproduces how often: 

100%

## Brave version (about:brave info) 

Brave | 0.54.4 Chromium: 70.0.3528.4 (Official Build) dev (64-bit)
-- | --
Revision | 1911f781145b803e04f2d0b5b1a0146ae69fcbdc-refs/branch-heads/3528@{#9}
OS | Windows

### Reproducible on current release:

N/A

### Website problems only:  
- Does the issue resolve itself when disabling Brave Shields? 
- Is the issue reproducible on the latest version of Chrome? 

### Additional Information

cc: @Snuupy @bbondy @bradleyrichter

**Comment by Snuupy** [comment]
possible reword ("clear + reload" in context menu)

**Comment by bsclifton** [comment]
Closing as `wontfix` - we have a new cosmetic filter in place. I created https://github.com/brave/brave-browser/issues/8914 to track removing this older version
